Landing page/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="header">
+      Microverse Supermarket
+      <img
+        src="https://i.pinimg.com/600x315/76/1b/a3/761ba3e1ddf1e70f4cb06906d5935668.jpg"
+        id="header-img"
+        alt="A supermarket clip art"
+      />
+      <br />
+      <br />
+      <br />
+      <br />
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#home">Home Page</a></li>
+          <li><a class="nav-link" href="#about">About</a></li>
+          <li><a class="nav-link" href="#contacts">Contacts</a></li>
+        </ul>
+      </nav>
```

Comment:
please use flex-box or grid to align links and ensure that they are part of the header. Also remove bullets from the link tags.